tests/performance/checkboxTreeView/.classpath

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
minor: should this project be moved to manual tests hierarchy?

BTW, I am planning to create a monkey tester application which will include many tests, to simplify the testing/verification of fixes JDK-8299335
This code be added to the tester, either as a separate page, or as a data source option for one of the TreeTableView page.

Reply:
I wasn't sure exactly where to put it, but it's going to be used in a fix for performance issue I found when a node has many children.

If it will fit into a grater app I don't mind.

Reply:
I'll move it there as a part of JDK-8299335

Reply:
I think it's fine as a separate test program for now. Once we have a tester application, we can consider which existing tests should go there.